### ISSUE TITLE
Thank you team for the excellent work, I try to reproduce your results, but I only have 26 on psnr, my cofigs are as follows, do you have any suggestions to improve my on psnr? I see that you can reach 36 in your paper

### ISSUE THREAD

**Issue by lll123-cpu** [NarcissusEx/CuNeRF#15]
My data shape is 512*512*D   CT（Lung window cross-section）  nii.gz

# example config for CuNeRF

# training
max_iter : 400000
eval_iter: 10000
save_iter: $eval_iter
log_iter : 100
bs_train : 384 # batch size
bs_eval  : 4096
bs_test  : $bs_eval
N_eval   : 50
lr       : 0.002
lr_final : 0.00002
workers  : 4

# modeling
stype     : cube
radius    : 1
n_samples : 64
n_imps    : 64
mtype     : NeRFMLP

# test
zpos      : [0]
scales    : [2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0]
angles    : [0]
asteps    : 45
axis      : [0, 1, 1]
cam_scale : 1.5

# path
result_path : /home/h3c/lmt/code/CuNeRF-main/save/results
save_path   : /home/h3c/lmt/code/CuNeRF-main/save/savepic
log_path    : /home/h3c/lmt/code/CuNeRF-main/save/logs

loss : 
  name : Adaptive_MSE_LOSS

sampling : 
  stype     : $stype
  n_samples : $n_samples

rendering :
  stype : $stype

importance :
  stype     : $stype
  n_samples : $n_imps

model : 
  name    : $mtype
  netD    : 8
  netW    : 256
  in_ch   : 3
  out_ch  : 2
  skips   : [4]
  max_freq: 10
  p_fns   : ['sin', 'cos']

model_ft : $model

dataset : 
  mode : $mode
  dname: Medical3D
  train : 
    file    : $file
    scale   : $scale
    radius  : $radius
    bsize   : $bs_train
    modality: $modality 

  eval :
    file    : $file
    scale   : $scale
    radius  : $radius
    N_eval  : $N_eval
    bsize   : $bs_eval
    modality: $modality 

  test :
    file      : $file
    scales    : $scales
    radius    : $radius
    axis      : $axis
    angles    : $angles
    zpos      : $zpos
    cam_scale : $cam_scale
    asteps    : $asteps

dataloader :
  mode : $mode
  train :
    batch_size : 1
    shuffle : True
  
  eval:
    batch_size : 1
    shuffle : False

  test:
    batch_size : 1
    shuffle : False

optim :
  name : Adam
  lr   : $lr
  betas: [0.9, 0.999]
  eps  : 0.000001

metrics : 
  psnr  : True
  ssim  : True
  lpips : False
  avg   : False

lr_decay : 
  lr_init        : $lr
  lr_final       : $lr_final
  max_iter       : $max_iter
  lr_delay_steps : 1000
  lr_delay_mult  : 0.01

clip_grad :
  max_val  : 0.1
  max_norm : 0.1

**Comment by djx99** [comment]
have you solved?

**Comment by djx99** [comment]
Loading model from: /home/mul/anaconda3/envs/lee_son_code/lib/python3.9/site-packages/lpips/weights/v0.1/alex.pth
max_iter        : 200000
eval_iter       : 10000
save_iter       : 10000
log_iter        : 100
bs_train        : 1024
bs_eval         : 32768
bs_test         : 32768
N_eval          : 50
lr              : 0.002
lr_final        : 2e-05
workers         : 4
stype           : cube
radius          : 1
n_samples       : 64
n_imps          : 64
mtype           : NeRFMLP
zpos            : [0]
scales          : [1]
angles          : [0]
asteps          : 45
axis            : [0, 1, 1]
cam_scale       : 1.5
result_path     : save/CuNeRFx1/liver_choose_1-50
save_path       : save/CuNeRFx1/liver_choose_1-50
log_path        : save/CuNeRFx1/liver_choose_1-50
loss
  name            : Adaptive_MSE_LOSS
----------------------------------------------------------
sampling
  stype           : cube
  n_samples       : 64
----------------------------------------------------------
rendering
  stype           : cube
----------------------------------------------------------
importance
  stype           : cube
  n_samples       : 64
----------------------------------------------------------
model
  name            : NeRFMLP
  netD            : 8
  netW            : 256
  in_ch           : 3
  out_ch          : 2
  skips           : [4]
  max_freq        : 10
  p_fns           : ['sin', 'cos']
----------------------------------------------------------
model_ft
  name            : NeRFMLP
  netD            : 8
  netW            : 256
  in_ch           : 3
  out_ch          : 2
  skips           : [4]
  max_freq        : 10
  p_fns           : ['sin', 'cos']
----------------------------------------------------------
dataset
  mode            : train
  dname           : Medical3D
  train           : {'file': 'data/liver_choose_1-50.nii.gz', 'scale': 1, 'radius': 1, 'bsize': 1024, 'modality': None}
  eval            : {'file': 'data/liver_choose_1-50.nii.gz', 'scale': 1, 'radius': 1, 'N_eval': 50, 'bsize': 32768, 'modality': None}
  test            : {'file': 'data/liver_choose_1-50.nii.gz', 'scales': [1], 'radius': 1, 'axis': [0, 1, 1], 'angles': [0], 'zpos': [0], 'cam_scale': 1.5, 'asteps': 45}
----------------------------------------------------------
dataloader
  mode            : train
  train           : {'batch_size': 1, 'shuffle': True}
  eval            : {'batch_size': 1, 'shuffle': False}
  test            : {'batch_size': 1, 'shuffle': False}
----------------------------------------------------------
optim
  name            : Adam
  lr              : 0.002
  betas           : [0.9, 0.999]
  eps             : 1e-06
----------------------------------------------------------
metrics
  psnr            : True
  ssim            : True
  lpips           : False
  avg             : False
----------------------------------------------------------
lr_decay
  lr_init         : 0.002
  lr_final        : 2e-05
  max_iter        : 200000
  lr_delay_steps  : 1000
  lr_delay_mult   : 0.01
----------------------------------------------------------
clip_grad
  max_val         : 0.1
  max_norm        : 0.1
----------------------------------------------------------
expname         : CuNeRFx1
cfg             : configs/example.yaml
scale           : 1
mode            : train
file            : data/liver_choose_1-50.nii.gz
save_map        : True
resume          : True
resume_type     : current
is_details      : True
is_gif          : False
is_video        : False
modality        : None

Resuming from save/CuNeRFx1/liver_choose_1-50/current.pkl
Failing in resuming from save/CuNeRFx1/liver_choose_1-50/current.pkl
  0%|                                                                                                                                                                   | 1/200000 [00:00<?, ?it/s]
[TRAIN] Iter: 10000 LR : 0.001715  LOSS : 0.000503  PSNR : 33.935545:   5%|███▉                                                                           | 10000/200000 [05:01<1:35:29, 33.16it/s]Saving checkpoint at save/CuNeRFx1/liver_choose_1-50/current.pkl
[EVAL] : 49: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 50/50 [02:29<00:00,  3.00s/it]
[BEST] psnr : 26.20625455235359 | ssim : 0.8397008776664734 | lpips : 0.2347143143415451 | avg : 0.06083122525019683███████████████████████████████████████████████| 50/50 [02:29<00:00,  2.99s/it]
[EVAL] PSNR : 26.20625455235359 | SSIM : 0.8397008776664734 | LPIPS : 0.2347143143415451 | AVG : 0.06083122525019683
[TRAIN] Iter: 20000 LR : 0.001567  LOSS : 0.000294  PSNR : 35.973539:  10%|███████▉                                                                       | 20000/200000 [12:40<1:31:01, 32.96it/s]Saving checkpoint at save/CuNeRFx1/liver_choose_1-50/current.pkl
[EVAL] : 49: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 50/50 [02:30<00:00,  3.00s/it]
[BEST] psnr : 26.20625455235359 | ssim : 0.8587431907653809 | lpips : 0.2015404999256134 | avg : 0.06011395948083104███████████████████████████████████████████████| 50/50 [02:30<00:00,  3.00s/it]
[EVAL] PSNR : 25.424410251404193 | SSIM : 0.8587431907653809 | LPIPS : 0.2015404999256134 | AVG : 0.06011395948083104
